jspoon/src/main/java/pl/droidsonroids/jspoon/HtmlField.java

```diff
@@ -38,7 +38,7 @@
 
     private void setLocaleFromTag(String localeTag) {
         if (localeTag.equals(Selector.NO_VALUE)) {
-            locale = Locale.getDefault();
+            locale = Locale.ROOT;
```

Comment:
Jeżeli chcę mieć w obrębie klasy testowej ten sam `default`, to wystarczy, że użyję `setDefault` w `@BeforeClass`, czy muszę jeszcze przywrócić w `@AfterClass`?